coprocess.go

```diff
@@ -102,7 +102,7 @@ func (c *CoProcessor) ObjectFromRequest(r *http.Request) *coprocess.Object {
 	if r.Body != nil {
 		defer r.Body.Close()
 		miniRequestObject.RawBody, _ = ioutil.ReadAll(r.Body)
```

Comment:
worth to catch and log error here

Reply:
Have extended `ObjectFromRequest` to also return errors 👍